mesonbuild/interpreter/dependencyfallbacks.py

```diff
@@ -340,6 +340,23 @@ def lookup(self, kwargs: TYPE_nkwargs, force_fallback: bool = False) -> Dependen
                         self._subproject_impl(subp_name, varname)
                     break
 
+                # Check if we could have a matching Rust crate imported from
+                # Cargo.lock, with expected api version. We know there is no
+                # wrap-file defined to match this crate.
+                if name.endswith('-rs') and required:
+                    s = name.split('-')
+                    if len(s) < 3:
+                        continue
+                    s.pop() # -rs
+                    s.pop() # version
+                    crate = '-'.join(s)
+                    cargo_wraps = self.wrap_resolver.get_cargo_wraps(crate)
+                    if not cargo_wraps:
+                        continue
+                    raise DependencyException(f'Rust dependency "{name}" not found, ' +
+                                              'but following versions were found using Cargo.lock import: ' +
+                                              ', '.join(['"' + n + '"' for n in cargo_wraps]))
```

Comment:
`f'"{n}"'` is more idiomatic than `'"' + n + '"'`

Reply:
Changed to this.